.changeset/great-deer-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@getodk/web-forms': minor
+---
+
+Render groups without a ref attribute
```

Comment:
There's now also a change to the engine that would be helpful to document as a minor change (because we're pre-1.0)?